### PR TITLE
DEC 513 - First search api usage.

### PR DESCRIPTION
**Pull request by danthewolfe**
* Create search display page based on items listing page.
* Move some pieces out of the item list page into separate component so they can be reusable (and used in new search page).
* Comment out some of the modal stuff that was broken in api changes from ndlib/honeycomb#187

### CODE REVIEW

**Comment by jonhartzler** [comment]
:+1:

File: Gemfile.lock [hunk 1]
```diff
@@ -388,6 +388,3 @@ DEPENDENCIES
   thin
   uglifier (>= 1.3.0)
   web-console (~> 2.0)
-
-BUNDLED WITH
-   1.10.6
```
Comment (jaronkk) [inline Gemfile.lock]: Can you update your bundler version?  `gem install bundler`, then run `bundle install`.  I want to make sure we don't keep getting this added and removed.

Comment (danthewolfe) [inline Gemfile.lock]: updated

**Comment by jaronkk** [comment]
:+1:

**Pull request merged**